package.json

```diff
@@ -52,6 +52,7 @@
     "eslint-config-airbnb-base": "^14.0.0",
     "eslint-plugin-import": "^2.19.1",
     "jest": "^24.9.0",
+    "lint": "^0.7.0",
```

Comment:
Did you accidentally `npm install lint` instead of `npm run lint` or something? This pulled in a bunch of crap from npm. If we're not using this, can you remove it and regenerate the package-lock please?